static/js/hotkey.js

```diff
@@ -1,5 +1,6 @@
 "use strict";
 
+const status = require("../js/user_status_ui");
```

Comment:
We do nor require this, you can directly use `user_status_ui.submit_new_status` when required.